llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -270,8 +285,26 @@ int main(int argc, char *argv[]) {
     MSG("\n");
   }
 
-  if (ComputeStats)
-    computeStats(*CAS, SummaryIDs);
+  if (ComputeStats.empty()) {
+    return 0;
+  }
+
+  bool PrintToStdout = false;
+  if (StringRef(ComputeStats) == "-")
+    PrintToStdout = true;
+
+  Optional<raw_fd_ostream> CsvStream;
```

Comment:
This is used not just for `.csv` files so it should be called something else, such as `StatsFile`.